docs/getting-started/getting-started-with-low-code-airflow.md

```diff
@@ -0,0 +1,168 @@
+
+A Job is an entity that contains Gems to represent a DAG consisting of various Tasks (Pipelines/Models/Scripts, etc) which you can Run once or schedule to run at a frequency. Each Job would represent an Airflow DAG in Python.
+
+Let's see how to create an Airflow Job in Prophecy.
+
+![Create_job](img/Airflow_2.1_Create_Job.png)
+
+Click the **(1) Create Entity** button, and choose **(2) Create Job** option.
+
+On the first screen, you would provide the Basic Info of the Job.
+
+In **(1) Project**, select the Project in which you want to create the Job. You can pick the existing Databricks Spark or SQL project here where you have created Pipelines/Models.
+In **(2) Branch**, Pick a development branch for the new Job. Here you can pick an existing branch for development, or create a new one.
+In **(3) Name**, provide a name for your Job.
+In **(4) Scheduler**, pick the type as **Airflow**.
+In **(5) Fabric**, Select the Fabric we created in previous step.
+In **(6) Schedule**, Pick a schedule with which you want to schedule the Job. Please note, you can modify this again after testing before releasing your Job.
+In **(7) Description**, add a description about the Job you are creating.
+
+Once done, click **(8) Create New**.
+
+This will take you the Job editor page where you would be creating the actual DAG for the Job.
+Let's start adding Pipelines/Models to our Job now.
+
+### 2.1 Adding Spark Pipeline Gem For Databricks to your DAG
+
+![Add_Pipeline_Gem](img/Airflow_2.2_Add_Pipeline_Gem.png)
+
+Click on **(1) Operators**, and Drag the **(2) Pipeline Gem** from the dropdown to the canvas. Then click the newly added Gem and click **(3) Open** to open the Gem Configurations.
+
+Here, you will select the Pipeline and Optionally override any config values for the Pipeline if you want.
+
+![Pipeline_Gem_Configurations](img/Airflow_2.3_Pipeline_Gem_Configurations.png)
+In **(1) Pipeline to Schedule** , select the Pipeline you want to Run. As you select the Pipeline, You would start seeing the Configurations defined in the Pipeline. You Would not be able to modify the schema of these configs but can override the Config values.
+
+In (**2) Fabric and Cluster size to run this Pipeline**, pick the Fabric and Job size for running this Pipeline in Databricks. Once done, Click **(3) Save**!!
+
+### 2.2 Adding SQL DBT Gem For Databricks to your DAG
+
+![Add_DBT_Gem](img/Airflow_2.4_Add_DBT_Gem.png)
+
+Click on **(1) Operators**, and Drag the **(2) DBT Gem** from the dropdown to the canvas. Then click the newly added Gem and click **(3) Open** to open the Gem Configurations.
+
+Here, we will select the DBT Project/Model to Schedule, what SQL Fabric to schedule it on, and other additional properties for running a DBT model.
+
+![DBT_Gem_Configurations](img/Airflow_2.5_DBT_Gem_Configurations.png)
+
+In **(1) DBT commands** , select the commands you want to run when scheduling your Models. You can select all (Deps, Seed, Run and Test) here.
```

Comment:
I looked to link "test" to a doc page, but I think "test" for SQL pipelines is just the interactive running of the pipeline, rather than running unit tests, right? Do we offer unit testing for DBT SQL models or jobs running models?